src/components/CohortBuilder/Summary/Cards/StudiesChart.js

```diff
@@ -138,13 +160,15 @@ class StudiesChart extends React.Component {
           <HorizontalBar
             showCursor={true}
             data={data}
+            tooltipDictionary={studies.map((s) => ({ label: s.code, tooltip: s.name }))}
             indexBy="label"
             keys={['probands', 'familyMembers']}
             tooltipFormatter={studiesToolTip}
             sortBy={sortDescParticipant}
             tickInterval={4}
             colors={getCohortBarColors(data, theme)}
             xTickTextLength={28}
+            xOffset={100}
```

Comment:
Magic number but it no big deal

Reply:
yeah... but it would be very hard to fix.